src/common/SurgeSynthesizerIO.cpp

```diff
@@ -609,7 +609,7 @@ void SurgeSynthesizer::savePatchToPath(fs::path filename, bool refreshPatchList)
     fxp.numPrograms = vt_write_int32BE(1);
     fxp.version = vt_write_int32BE(1);
     fxp.fxVersion = vt_write_int32BE(1);
-    strncpy(fxp.prgName, storage.getPatch().name.c_str(), 28);
```

Comment:
didn't you reverse the order here? strncpy has arguments "destination source length" so this means "fxp.prgName = storage.getPatch().name"

Reply:
Whoops, reverting.